src/app/components/annotations/Annotation.js

```diff
@@ -5,8 +5,6 @@ import Relay from 'react-relay/classic';
 import RCTooltip from 'rc-tooltip';
 import styled from 'styled-components';
 import { stripUnit } from 'polished';
-import Lightbox from 'react-image-lightbox';
```

Comment:
This is great, all that's left is `AspectRatio` and we can get rid of this lightbox dependency at some point...